entity-service-impl/src/test/java/org/hypertrace/entity/query/service/converter/ValueHelperTest.java

```diff
@@ -96,17 +103,32 @@ void testIsPrimitiveForAllNonPrimitive(final ValueType type) {
   class ConvertToSubDocumentValueTest {
     @Test
     void testConvertStringValueToSubDocumentValue() throws ConversionException, IOException {
+      final String columnName = "string";
+      when(attributeMapping.getAttributeKind(context, columnName))
+          .thenReturn(Optional.of(AttributeKind.TYPE_STRING));
+      when(attributeMapping.isPrimitive(AttributeKind.TYPE_STRING)).thenReturn(true);
       assertEquals(
-          SubDocumentValue.of(new JSONDocument("{ \"value\": {\"string\": \"Mars\" }}")),
+          SubDocumentValue.of(
+              new JSONDocument("{ \"value\": {\"" + columnName + "\": \"Mars\" }}")),
```

Comment:
This seems off. We always store in the format `{ "value": { <data_type>: <value> }` for primitive types.

Reply:
Just replaced `string` with a variable as it was getting used multiple times. Rename variable from `columnName` to `dataType` for better readability.